CONTRIBUTING.md

```
@@ -0,0 +1,108 @@
+
+This compiles TypeScript to `dist/` and generates type declarations.
+
+### Test
+
+```bash
+pnpm test
+```
+
+### Project Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)